lib/facturacr/xml_document.rb

```diff
@@ -105,7 +105,9 @@ def initialize(xml_provider)
           if @document.version_42?
             item.code = line.css("Codigo Codigo").text
           elsif @document.version_43?
-            item.code = line.css("CodigoComercial Codigo").text
+            code = line > "Codigo"
```

Comment:
use single quotes on simple string
'CodigoComercial Codigo'

in general all file